### PR TITLE
docs: added basic Slack documentation

### PR DESCRIPTION
**Pull request by UlisesGascon**

### CODE REVIEW

File: how-to/manage-slack.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Slack 
+
+We use slack to discuss about the projects and coordinate activities related to them. We follow [the same CoC](https://github.com/onebeyond/admin/issues/4) as in Github.
+
+Right now the slack is use to improve the communication between the maintainers. If you need access please create an issue in this repo
```
Comment (inigomarquinez) [inline how-to/manage-slack.md:5]:
```suggestion
Right now, slack is used to improve the communication between the maintainers. If you need access, please create an issue in this repo.
```

Apart from the suggestion, the link to the repo is missing.

File: how-to/manage-slack.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Channels
+
+- `#00-announcements`: Important news 
+- `#01-general`: General channel great for starting discussions
+- `#03-ask-anything`: Great if you are lost
```
Comment (inigomarquinez) [inline how-to/manage-slack.md:11]: What happens with `02`? 🤔

File: how-to/manage-slack.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- `#04-infra`: The place to notify and track issues with CI, tokens, accesses...
+- `#05-notifications`: Automatic notifications from repositories, etc...
+
+There are other channels that are specifics to projects like `#monaco`, `#systemic`...
+
+As well there are some private channels like `#moderation`, `#admins`, etc..
```
Comment (inigomarquinez) [inline how-to/manage-slack.md:17]: Should we divide them in bullet points, similar to the public ones, and add a brief description?

**Review by inigomarquinez — CHANGES_REQUESTED** [review]
Some suggestions added

File: how-to/manage-slack.md (hunk 2 above)
Comment (UlisesGascon) [reply how-to/manage-slack.md]: Great catch! Solved by adding `#02-random` in [2d85221](https://github.com/onebeyond/admin/pull/23/commits/2d85221b3d40fc072d9315164f82d3dfa088f407)

File: how-to/manage-slack.md (hunk 3 above)
Comment (UlisesGascon) [reply how-to/manage-slack.md]: Refactored in [2d85221](https://github.com/onebeyond/admin/pull/23/commits/2d85221b3d40fc072d9315164f82d3dfa088f407)

**Review by inigomarquinez — APPROVED** [review]
Looks great!

**Pull request merged**